Makefile

```
@@ -0,0 +1,122 @@
+PROJECT_NAME := Pulumi Xyz Resource Provider
+
+PACK             := xyz
+PACKDIR          := sdk
+PROJECT          := github.com/pulumi/pulumi-xyz
+NODE_MODULE_NAME := @pulumi/xyz
+NUGET_PKG_NAME   := Pulumi.xyz
+
+PROVIDER        := pulumi-resource-${PACK}
+CODEGEN         := pulumi-gen-${PACK}
+VERSION         ?= $(shell pulumictl get version)
+PROVIDER_PATH   := provider/v3
```

A brand-new provider shouldn't start with v3
```suggestion
PROVIDER_PATH   := provider
```

---

fixed